oar/cli/cmd_change_advisory_status.py

```diff
@@ -30,6 +30,9 @@ def change_advisory_status(ctx, status):
         am = AdvisoryManager(cs)
         # update task status to in progress
         report.update_task_status(LABEL_TASK_CHANGE_AD_STATUS, TASK_STATUS_INPROGRESS)
+        # check kernel tag before change advisories' status
+        if am.check_kernel_tag():
```

Comment:
`check_kernel_tag ` is not func of `AdvisoryManager ` need to filter out image advisory and call `check_kernel_tag` then

Reply:
if kernel tag is found, we can raise `AdvisoryException`